src/main/content/_includes/doc_header.html

```diff
@@ -21,23 +21,31 @@
                         <a href="/guides">Guides</a>
                     </li>
                     <li>
-                        <a href="/docs/ref/javaee/7"> Java EE 7 API</a>
+                        <a href="/docs/ref/javaee/7">Java EE 7 API</a>
                     </li>
                     <li>
                         <a href="/docs/ref/microprofile/1.2">MicroProfile 1.2 API</a>
-                    </li>
+                    </li>                    
                 </ul>
             </div>
             <div id="breadcrumb_row">
-                <ol class="breadcrumb fluid-container">
-                    <li>
-                        <a href="/docs">Docs and Guides</a>
-                    </li>
+                <ol class="breadcrumb fluid-container"> 
+                    {% for url in page.breadcrumb %}
+                    {% assign breadcrumb_page = site.pages | where: 'permalink', url %}
+                    {% assign breadcrumb_page = breadcrumb_page[0] %}
+                        <li>
+                            <a href="{{ breadcrumb_page.url }}"> {{ breadcrumb_page.title }}</a>
+                        </li>
+                    {% endfor %} 
                     <li class="active"> {{ page.title }} </li>
                 </ol>
             </div>
         </div>
     </nav>
+    <div id="background_container" class="black_blue_gradient_background">
+            <div id="docs_container" class="container">
```

Comment:
Do we need `#docs_container` in the header?  If we do need `#docs_container`, I think it would be more appropriate for this to be in the _layout

Reply:
I recall we only need `#background_container` so that we get the `black_blue_gradient_background` for the background of the header.  We may even be able to rename `#background_container` to just `#header_background`.  I think "container" was once referring to an actual bootstrap container, which we are no longer using (I think?)

Reply:
#background_container is used on every other page, but we can rename it to header_background for every docs page. Check the latest commit.